src/frontend.c

```diff
@@ -73,6 +74,13 @@ int main(int argc, char *argv[]) {
     setup_colors();
   }
 
+  // enable mouse input
+  mmask_t return_mask =
+      mousemask(ALL_MOUSE_EVENTS | REPORT_MOUSE_POSITION | BUTTON1_PRESSED |
+                    BUTTON1_RELEASED | BUTTON1_CLICKED,
+                NULL);
+  logd("Returned mouse mask: %li\n", return_mask);
```

Comment:
This is failing on Travis b/c return_mask is interpretted as an "unknown variable" when DEBUG isn't set (and the `logd` macro is expanded to nothing).

Reply:
Resolved with a technique from [`zf_log`](https://github.com/wonder-mice/zf_log).